assets/scss/_download.scss

```diff
@@ -0,0 +1,19 @@
+.download {
```

Comment:
I know there was no design for mobile and tablets, that is why it need a change :P 

right now it looks like this:
![Screenshot from 2020-03-31 23-52-20](https://user-images.githubusercontent.com/25872895/78079657-d4cd1280-73ac-11ea-8f5a-73f6844f0007.png)

and that it why we are using "mobile-first" approach -> we develop and display things that are needed and on the bigger screens like tablet and later on desktop we give all the stuff, animations, colorful effects etc.
so please for mobile set title and button centered (by flex) and give display none for the image:
![Screenshot from 2020-03-31 23-56-20](https://user-images.githubusercontent.com/25872895/78079813-31c8c880-73ad-11ea-8851-877446eaaba7.png)

and for the tablets, try something like this, it doesn't need to be perfect:
![Screenshot from 2020-04-01 00-01-14](https://user-images.githubusercontent.com/25872895/78079940-75233700-73ad-11ea-8e9e-f12faed45400.png)

also there is no space between sections features and download -please give it some margin :)